### PR TITLE
Add retries to important scripts

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

**Comment by eherot** [comment]
@jeffbyrnes you should still review this PR but I am merging it right now so that we can hopefully have a more reliable repair script over the weekend.

File: .kitchen.cloud.yml [hunk 1]
```diff
@@ -41,6 +41,10 @@ suites:
       et_cassandra:
         snapshot:
           data_bag: false
+          wait_secs: 2
+          tar_retry_secs: 0
+        incrementals:
+          upload_retry_delay: 0
         snapshot_conf:
           SKIP_KEYSPACES: (system_traces OpsCenter)
```
Comment (jeffbyrnes) [inline .kitchen.cloud.yml]: @eherot unrelated, but the cloud & vagrant attributes do not match w/r/t the `SKIP_KEYSPACES` attribute. In one case, it’s under `[et_cassandra][snapshot]`, the other, `[et_cassandra][snapshot_conf]`. Something to fix elsewhere, perhaps?

File: templates/default/upload-incrementals.erb [hunk 2]
```diff
@@ -8,16 +8,18 @@ set -e
...
 
+let upload_retries=5
+
 root_dir=`dirname "<%= node['et_cassandra']['config']['data_file_directories'].first %>"`
 data_dirs="<%= node['et_cassandra']['config']['data_file_directories'].join(' ') %>"
 work_dir="${root_dir}/backup_work_dir"
 old_pwd="$PWD"
 
-root_dir_fs_id=`/usr/bin/stat -f -c %i ${root_dir}`
+work_dir_fs_id=`stat -f -c %i ${work_dir}`
```
Comment (jeffbyrnes) [inline templates/default/upload-incrementals.erb]: No need for the absolute path for `stat`?

**Comment by jeffbyrnes** [comment]
@eherot looks pretty legit. A few things I noticed.

**Pull request merged**